cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -9,10 +9,16 @@ KUBECONFIG_PATH="${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig"
 
 MASTER_NODE="${CLUSTER_NAME}-control-plane"
 WORKER_NODE_ROOT="${CLUSTER_NAME}-worker"
+PFS_COUNT_PER_NODE=${PFS_COUNT_PER_NODE:-1}
 
 OPERATOR_GIT_HASH=8d3c30de8ec5a9a0c9eeb84ea0aa16ba2395cd68  # release-4.4
 SRIOV_OPERATOR_NAMESPACE="sriov-network-operator"
 
+if [ $PFS_COUNT_PER_NODE -le 0 ]; then
+  echo "FATAL: Wrong value of PFS_COUNT_PER_NODE, must be a positive integer, less or equal to the actual number of PFs available"
+  exit 1
+fi
```

Comment:
Please drop the `if`, there is no need for it.
The message is also too much IMO, just saying that it needs to be a positive integer should be enough.
The "the actual number of PFs available" is also redundant, if one specifies too many, then a different error should pop up.